concent_api/generate-requirements-lock.sh

```diff
@@ -2,9 +2,21 @@
 
 if [[ "$VIRTUAL_ENV" != "" ]]
 then
-    echo "--find-links https://builds.golem.network/simple/pyelliptic" > ${BASH_SOURCE%/*}/requirements.lock
-    pip freeze >> ${BASH_SOURCE%/*}/requirements.lock
-    echo "Done"
+    if [ "$#" -ne 1 ]
+    then
+        echo "Illegal number of parameters"
+        exit 1
+    else
+        if ! [[ -f $1 ]]
```

Comment:
Please add variable to first argument at the beginning of the file. variable=$1

Reply:
This statement is not correct. It should contain single square brackets. Also, you checked wrong condition. This should look like this 
```bash
if [ -f $1 ]
```

Reply:
also, use option `-s` which is check that file is not empty.